src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -559,8 +559,12 @@
   <data name="Convert_to_raw_string" xml:space="preserve">
     <value>Convert to raw string</value>
   </data>
-  <data name="Convert_to_raw_string_no_indent" xml:space="preserve">
-    <value>Convert to raw string (no indent)</value>
+  <data name="without_leading_whitespace_may_change_semantics" xml:space="preserve">
+    <value>... without leading whitespace (may change semantics)</value>
+    <comment>This clause is a follow up to the "Convert to raw string" loc string.
+    The intent is that the user sees "Convert to raw string" as an option to select,
+    and that is then followed with this clause.  This is so we don't have a huge string
+    saying "Convert to raw string without leading whitespace (may change semantics)"</comment>
```

Comment:
i do think it's important (personally).  Otherwise, i worry that someone may be confused about waht "indentation" means.  e.g. if it's the content indentation that is adjusted, or the *syntax* indentation is adjusted.